docs/upgrade.md

```diff
@@ -112,6 +112,16 @@ environment variable.
 See [using a forward proxy with Synapse documentation](setup/forward_proxy.md) for
 details.
 
+## UIA fallback templates can display errors
+
+This may affect you if you make use of custom HTML templates for the
+[reCAPTCHA](../synapse/res/templates/recaptcha.html) and
+[terms](../synapse/res/templates/terms.html) fallback pages.
+There is no configuration option for these templates.
```

Comment:
I'm not sure what this means -- what would the configuration option do?

Reply:
Well for other templates (e.g. account validity) there are configuration options so you can give a path to custom templates. I'll just remove that line though because it's probably just confusing.